armi/utils/__init__.py

```diff
@@ -52,18 +52,25 @@
 
 def coverageReportHelper(config, dataPaths):
     """
-    Small utility function to generate converage reports.
+    Small utility function to generate coverage reports.
 
     This was created to side-step the difficulties in submitting multi-line python
     commands on-the-fly.
+
+    This combines data paths and then makes html and xml reports for the
+    fully-combined result.
     """
     from coverage import Coverage
     import coverage
 
     try:
         cov = Coverage(config_file=config)
-        cov.combine(data_paths=dataPaths)
-        cov.save()
+        if dataPaths:
+            # fun fact: if you combine when there's only one file, it gets deleted.
```

Comment:
Is this worth submitting an issue upstream?

Reply:
It's at least related to this one: https://github.com/nedbat/coveragepy/issues/516

I guess their solution was to add the `strict` option... ok. Our solution here still makes sense for our usage.